t/unit-tests/u-urlmatch-normalization.c

```diff
@@ -1,21 +1,21 @@
-#include "test-lib.h"
+#include "unit-test.h"
 #include "urlmatch.h"
 
 static void check_url_normalizable(const char *url, unsigned int normalizable)
 {
 	char *url_norm = url_normalize(url, NULL);
 
-	if (!check_int(normalizable, ==, url_norm ? 1 : 0))
-		test_msg("input url: %s", url);
+	if ((normalizable ? 1 : 0) != (url_norm ? 1 : 0))
+		cl_failf("input url: %s", url);
```

Comment:
We're losing information around what the respective values are that are mismatching. The same is also true for all the other cases. So I'd either:

- Use `cl_assert_equal_i()` directly.
- Or adapt the `cl_failf()` messages so that they print the actual and expected values.